etabotsite/templates/index.html

```diff
@@ -0,0 +1,23 @@
+{% load staticfiles %}
```

Comment:
Are you sure you want to add those `index.html` and the static files into pmp repo?

Reply:
yes, I think it is ok as an example, and it helps with our testing.